crates/db_schema/src/utils.rs

```diff
@@ -135,11 +135,7 @@ async fn build_db_pool_settings_opt(settings: Option<&Settings>) -> Result<DbPoo
   let db_url = get_database_url(settings);
   let pool_size = settings.map(|s| s.database.pool_size).unwrap_or(5);
   let manager = AsyncDieselConnectionManager::<AsyncPgConnection>::new(&db_url);
-  let pool = Pool::builder()
-    .max_size(pool_size)
-    .min_idle(Some(1))
-    .build(manager)
-    .await?;
+  let pool = Pool::builder(manager).max_size(pool_size).build()?;
```

Comment:
I think the min_idle was added for a reason so you should keep it.

Reply:
The managed deadpool doesn't seem to have idle. All it has is max_size, and timeouts: https://docs.rs/deadpool-diesel/0.4.1/deadpool_diesel/struct.PoolConfig.html .

deadpool also by default smartly sizes the pool size to the number of CPUs, which apparently is the optimal way to do it anyway, so we might even want to remove that `pool_size` setting at some point.

mobc also looks more developed than bb8, and has a few more tunable configs, like min_idle, but deadpool still looks a bit more developed and popular.